crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -409,32 +427,80 @@ fn entity_props_ui(
         });
 }
 
-fn visible_history_ui(
+/// These space views support the Visible History feature.
+static VISIBLE_HISTORY_SUPPORTED_SPACE_VIEWS: once_cell::sync::Lazy<HashSet<SpaceViewClassName>> =
+    once_cell::sync::Lazy::new(|| {
+        [
+            SpatialSpaceView3D::NAME,
+            SpatialSpaceView2D::NAME,
+            TimeSeriesSpaceView::NAME,
+        ]
+        .map(Into::into)
+        .into()
+    });
+
+/// Entities containing one of these components support the Visible History feature.
+static VISIBLE_HISTORY_COMPONENT_NAMES: once_cell::sync::Lazy<Vec<ComponentName>> =
```

Comment:
For consistency's sake:
```suggestion
static VISIBLE_HISTORY_SUPPORTED_COMPONENT_NAMES: once_cell::sync::Lazy<Vec<ComponentName>> =